build

```diff
@@ -89,6 +95,7 @@ make_relup() {
             fi
             releases+=( "$base_vsn" )
         done < <("$FIND" _upgrade_base -maxdepth 1 -name "${name_pattern}.zip" -type f)
```

Comment:
with the help of the db, we should be able to find the exact path/name of each upgrade base package,
so there should be no need for this FIND command anymore ?